test-docker-compose/main.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"fmt"
+	"math/big"
+	"net"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+func main() {
+	if err := makeCerts(); err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
```

Comment:
If this is where the CI test exits, then maybe print to Stdout instead to see what's causing it?